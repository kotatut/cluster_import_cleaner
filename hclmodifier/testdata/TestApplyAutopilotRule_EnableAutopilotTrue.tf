resource "google_container_cluster" "primary" {
  name               = "my-cluster"
  location           = "us-central1"
  enable_autopilot   = true
  node_pools {
    name = "default-node-pool"
  }
}
