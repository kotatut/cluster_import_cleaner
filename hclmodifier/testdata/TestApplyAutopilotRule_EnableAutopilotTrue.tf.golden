resource "google_container_cluster" "primary" {
  name               = "my-cluster"
  location           = "us-central1"
  enable_autopilot   = true
}
